### PR TITLE
Add Moyinoluwa Adeyemi

### LINKED ISSUE #130: Submit names of Nigerian githubbers that inspire me
There are so many women Nigerian Githubers building cool stuff, I will be adding them to the list

### PR DESCRIPTION
**Pull request by prondubuisi**
fix #130

### CODE REVIEW

**Review by tapaswenipathak — COMMENTED** [review]
Please say `fix #issue_number` in the next PR's. 

https://help.github.com/en/articles/closing-issues-using-keywords

File: README.md [hunk 1]
```diff
@@ -164,6 +164,8 @@ Mary Rose Cook ([maryrosecook](https://github.com/maryrosecook))
 
 Maxime Chevalier-Boisvert ([maximecb](https://github.com/maximecb))
 
+Moyinoluwa Adeyemi ([moyheen](https://github.com/moyheen))
```
Comment (tapaswenipathak) [inline README.md]: I think `o` is after `i` in alphabets?

Comment (prondubuisi) [reply README.md]: yes @tapaswenipathak. Made an update. Am sleepy already D:

Comment (tapaswenipathak) [reply README.md]: oh what timezone?

Comment (tapaswenipathak) [reply README.md]: oh np, what timezone?

Comment (prondubuisi) [reply README.md]: GMT +1 lol

**Pull request merged**